onnxruntime/core/common/path.cc

```diff
@@ -253,6 +253,21 @@ Path& Path::Append(const Path& other) {
   return *this;
 }
 
+Path& Path::Concat(const PathString& string) {
+  components_.back() += string;
```

Comment:
sorry i'm getting to this late. what if components_ is empty?

Reply:
Good catch. Fixed in https://github.com/microsoft/onnxruntime/pull/4525